libgalois/src/MemorySupervisor.cpp

```diff
@@ -17,18 +16,19 @@ const std::string oversubscribed_str = "memory manager oversubscribed";
 const std::string unregister_str = "memory manager unregister";
 
 void
-LogState(const std::string& str, count_t active, count_t standby) {
+LogState(const std::string& str, count_t standby, count_t bytes_reclaimed) {
   katana::GetTracer().GetActiveSpan().Log(
       str, {
-               {"active", active},
                {"standby", standby},
+               {"reclaimed", bytes_reclaimed},
            });
 }
 
 void
-KillCheck(katana::MemoryPolicy* policy, count_t active, count_t standby) {
-  if (policy->KillSelfForLackOfMemory(active, standby)) {
-    LogState(oversubscribed_str, active, standby);
+KillCheck(
```

Comment:
This name is a little confusing but I think it's better than renaming the function to `ShouldIKillMyself`

Reply:
Yeah, I hear you.  It is an internal function, along with `SanityCheck`.

Reply:
Fair enough